2_process/src/process_helpers.R

```diff
@@ -0,0 +1,13 @@
+
+get_group_ids <- function(group_info) {
+  sort(unique(group_info$group_id))
+}
+
+extract_morphometry <- function(config_fn) {
+  
+  config <- fromJSON(config_fn)
+  morphometry <- purrr::map(config, "morphometry")
```

Comment:
not a deal breaker at all, but just noting here that you are pulling out the whole morphometry block which includes some elements that aren't used in your depth/area functions. Things like lake_name, longitude, latitude, etc. Having these along don't hurt anything, _unless_ we make a change that is meaningless to the toha calculation, but changes these "morphometry" objects, such as updating all of the lake names, or changing the precision of the lat/lon values. This isn't a necessary change, just a heads up about isolating the thing you want to depend on (in this case `H` and `A` values) and nothing more if possible and practical.

Reply:
Gotcha - good point. We do actually need lat/lon for TOHA though so that the irradiance can be upsampled to minutely.

Reply:
But - I could easily keep those 4 things - `latitude`, `longitude`, `H`, and `A`

Reply:
yes, good call.